docs/UserGuide.md

```diff
@@ -192,15 +327,17 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd contact in your SoConnect.
 * `find Betsy` followed by `delete 1` deletes the 1st contact in the results of the `find` command.
 
-### Clearing all contacts : `clear`
+<br>
+
+#### 7.2.7. Clearing all contacts : `clear`
 
 Clears all contacts from your SoConnect.
 
 Format: `clear`
 
-## Tag Management Features
```

Comment:
Isn't Tag features of a different category than contact features?

Reply:
Please see the explanation below